proto/exocore/avs/v1/genesis.proto

```
@@ -0,0 +1,61 @@
+syntax = "proto3";
+package exocore.avs.v1;
+
+import "exocore/avs/v1/tx.proto";
```

buf

import "exocore/avs/v1/tx.proto": file does not exist

(COMPILE)